### ISSUE TITLE
Double category

### ISSUE THREAD

**Issue by l4l** [Allexin/TrackYourTime#6]
Create 2 category instead of one in applications menu.
ArchLinux x64, qt 5.5.1-9

Compiled with gcc 5.3.0-3:

``` bash
cd TrackYourTime/TrackYourTime
qmake && make
./TrackYourTime
```

**Comment by Allexin** [comment]
This bug fixed in source. Win32 version available for binary download

**Comment by l4l** [comment]
It can't be fixed yet. I build from last commit. If so may point out for it?
And did not understand about win32 version

**Comment by Allexin** [comment]
This bug - from two connect between m_CategoriesMenu and ApplicationsWindow in applicationswindow.cpp
Now second connect deleted and bug must be fixed.
This bug multiplatform - windows version now builded without this bug. other platforms - not.

**Comment by Allexin** [comment]
Fixed in 0.9.2 release
